src/core_plugins/kibana/public/dashboard/dashboard_actions.js

```diff
@@ -0,0 +1,121 @@
+import {
+  EMBEDDABLE_RENDER_FINISHED,
+  EMBEDDABLE_RENDER_REQUESTED,
+  EMBEDDABLE_RENDER_ERROR,
+  UPDATE_PANELS,
+  UPDATE_PANEL,
+  UPDATE_VIEW_MODE,
+  UPDATE_MAXIMIZED_PANEL_ID,
+  DELETE_PANEL,
+  UPDATE_IS_FULL_SCREEN_MODE,
+} from './action_types';
+
+function embeddableRenderRequested(panelId) {
+  return {
+    type: EMBEDDABLE_RENDER_REQUESTED,
+    panelId: panelId
```

Comment:
At the very least I think we should be using [Flux Standard Actions](https://github.com/acdlite/flux-standard-action#flux-standard-action) and not just some free-form action shape (read the readme, the library isn't a requirement).

The short version is that action objects only every have a `type`, `payload`, conditional `error`, and optional `meta` property.

```js
return {
    type: EMBEDDABLE_RENDER_REQUESTED,
    payload: {
        panelId: panelId
    }
}
```